lib/node_modules/@stdlib/blas/ext/base/ssumkbn2/manifest.json

```diff
@@ -1,40 +1,76 @@
 {
-	"options": {},
+	"options": {
```

Comment:
This file has mixed TAB and space indentation. It should be strictly 2-space indentation.